swig/include/python/gdal_python.i

```diff
@@ -2544,7 +2545,15 @@ def VectorTranslateOptions(options=None, format=None,
         and is only an afterwards conversion.
     explodeCollections:
         produce one feature for each geometry in any kind of geometry collection in the
-        source file, applied after any -sql option.
+        source file, applied after any -sql option. This option is not compatible with
```

Comment:
those pieces of doc should be pub in ogr2ogr.rst (in ogr2ogr.rst, the hint of using SQL like "SELECT fid AS original_fid, * FROM ..." can also be added if preservation of the FID is whished)

Reply:
should be put in ogr2ogr.rst *as well*